livre/eq/Config.cpp

```diff
@@ -158,7 +155,10 @@ void Config::unmapFrameData()
 
 void Config::resetCamera()
```

Comment:
function has a side effect ( function name does not tell the publishing ). resetCameraAndPublish() ?